api/app/Policies/PoolPolicy.php

```diff
@@ -123,6 +123,17 @@ public function updateDraft(User $user, Pool $pool)
             && $user->isAbleTo('update-team-draftPool', $pool->team);
     }
 
+    /**
+     * Determine whether the user can published draft pools.
+     *
+     * @return \Illuminate\Auth\Access\Response|bool
+     */
+    public function updatePublished(User $user, Pool $pool)
+    {
+        return $pool->getStatusAttribute() === PoolStatus::PUBLISHED->name
+            && $user->isAbleTo('update-any-publishedPool');
+    }
```

Comment:
@tristan-orourke something I noticed is that no one has the ability to edit any pool. Should I maybe make this `update-team-publishedPool` instead?

Reply:
I think Platform Admins and Community Managers should have `update-any-publishedPool`.